comprehensive/lms/templates/navigation.html

```diff
@@ -220,7 +220,9 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 <%include file="help_modal.html"/>
 ## Don't show cookie-banner if already accepted.
 <script type="text/javascript">
+
     if (cookieNotice.getCookie("cookie-banner") == "true") {
```

Comment:
good info for the triple equality!
the inlining was part of the earlier impl. so moved that logic to the JS file.
And yes, calling off multi times is a no-op.